api.c

```diff
@@ -87,11 +87,11 @@ uint32_t tcmu_cdb_get_xfer_length(uint8_t *cdb)
 	case 6:
 		return cdb[4];
 	case 10:
-		return be16toh(*((uint16_t *)&cdb[7]));
+		return get_unaligned_be16(&cdb[7]));
 	case 12:
-		return be32toh(*((u_int32_t *)&cdb[6]));
+		return get_unaligned_be32(&cdb[6]));
 	case 16:
-		return be32toh(*((u_int32_t *)&cdb[10]));
+		return get_unaligned_be32(&cdb[10]));
```

Comment:
Did you ever compile this change ? Please remove the last `)`.

Reply:
Now it compiles. Checked compilation.